test/features/kogito_service_performance.feature

```diff
@@ -148,8 +157,8 @@ Feature: Kogito Service Performance
       }
       """
 
-    Then Service "process-springboot-example" contains <requests> instances of process with name "orders" within 1 minutes
-    And Service "process-springboot-example" contains <requests> instances of process with name "orderItems" within 1 minutes
+    #Then Service "process-springboot-example" contains <requests> instances of process with name "orders" within 1 minutes
```

Comment:
same here

Reply:
See the point no. 4 in the PR description.